### ISSUE TITLE
Failed to parse level-type bigglobe:bigglobe, defaulting to minecraft:normal

### ISSUE THREAD

**Issue by UnknownDerpyPro** [Builderb0y/BigGlobe#42]
When hosting a **fabric** server for **1.21.1** (Using **MCSS**), setting the Level Type to: `level-type=bigglobe\:bigglobe`
I get an error message within my server console saying `Failed to parse level-type bigglobe:bigglobe, defaulting to minecraft:normal`
I checked config and its still the same, I added an extra `\` cause java is silly with backslashes (forgot what exactly java treats them as so I added another to show up, and it updated the Fail message to contain `bigglobe\:bigglobe` but config has been updated and now it has 3, so I deleted all of them, tried it out, didn't work and the config was updated with only one backslash. I am very confused.

Steps to re-create this error

1.  Create a fabric server using MCSS (Server Wrapper) [**Eclipse Adoptium jdk-21.0.4.7-hotspot**]
2.  Add the mods (Big Globe, and Fabric Api)
3.  Change the leveltype to what is mentioned within modrinth `level-type=bigglobe\:bigglobe`
4.  Witness the wackyness

-- Intentional Space --

Here is the exact logs in console
```
[15:04:44] [ServerMain/INFO]: Environment: Environment[sessionHost=https://sessionserver.mojang.com, servicesHost=https://api.minecraftservices.com, name=PROD]
[15:04:45] [ServerMain/INFO]: No existing world data, creating new world
[15:04:45] [ServerMain/WARN]: Failed to parse level-type bigglobe:bigglobe, defaulting to minecraft:normal
[15:04:46] [ServerMain/INFO]: Loaded 1290 recipes
[15:04:46] [ServerMain/INFO]: Loaded 1399 advancements
[15:04:47] [Server thread/INFO]: Starting minecraft server version 1.21.1
[15:04:47] [Server thread/INFO]: Loading properties
[15:04:47] [Server thread/INFO]: Default game type: SURVIVAL
[15:04:47] [Server thread/INFO]: Generating keypair
[15:04:47] [Server thread/INFO]: Starting Minecraft server on *:25566
[15:04:47] [Server thread/INFO]: Using default channel type
[15:04:47] [Server thread/INFO]: Preparing level "world"
[15:04:50] [Server thread/INFO]: Preparing start region for dimension minecraft:overworld
[15:04:50] [Worker-Main-3/INFO]: Preparing spawn area: 2%
[15:04:50] [Worker-Main-4/INFO]: Preparing spawn area: 2%
[15:04:51] [Worker-Main-12/INFO]: Preparing spawn area: 2%
[15:04:51] [Worker-Main-15/INFO]: Preparing spawn area: 18%
[15:04:52] [Worker-Main-15/INFO]: Preparing spawn area: 51%
[15:04:52] [Server thread/INFO]: Time elapsed: 2455 ms
[15:04:52] [Server thread/INFO]: Done (4.966s)! For help, type "help"
```

**Comment by Builderb0y** [comment]
Can you clarify what MCSS is? I definitely have gotten big globe to run on servers before. If I'm reading minecraft's source code correctly, the only way for that message to show up is if big globe's world preset isn't loaded at all, which then makes me question whether or not the rest of big globe is loaded. Can you upload the full server log as well? I want to see if big globe is saying it's loading or not.

**Comment by UnknownDerpyPro** [comment]
> Can you clarify what MCSS is? I definitely have gotten big globe to run on servers before. If I'm reading minecraft's source code correctly, the only way for that message to show up is if big globe's world preset isn't loaded at all, which then makes me question whether or not the rest of big globe is loaded. Can you upload the full server log as well? I want to see if big globe is saying it's loading or not.

@Builderb0y 

MCSS, MC Serrver Soft, is a Minecraft server wrapper; basically software that adds a GUI and easy navigation and management for server runtimes that don't have such GUIs (`https://www.mcserversoft.com/`)

The logs are the exact full logs, everything the console has made in the output; appears the mod does not load. In fact neither does Fabric API, thanks for noticing that.

I have checked MCSS as well as the file and realized its launching `server.jar` rather than `fabric-server-launch.jar`

I appologize for the missinformation and I have fixed it.

May this issue be deleted and removed as it is created from lack of awareness of mods even launching. Again I appologize for misunderstanding that the mods weren't loading at all.
